### PR TITLE
Combined imagenet and cifar-10 estimator tests

### PR DESCRIPTION
**Pull request by tfboyd**

### CODE REVIEW

**Comment by tfboyd** [comment]
Not ready for review yet.

**Comment by tfboyd** [comment]
Ready incase the last message was confusing  @reedwm

File: official/resnet/resnet_run_loop.py [hunk 1]
```diff
@@ -526,7 +526,7 @@ def resnet_main(
       This is only used if flags_obj.export_dir is passed.
 
   Returns:
-    Dict of results of the run.
+    Dict of results of the run. eval_results and train_hooks.
```
Comment (reedwm) [inline official/resnet/resnet_run_loop.py]: Clarify what this means. "Dict of results of the run, with keys 'eval_results' and 'train_hooks'". And similarly in imagenet_main.py.

File: official/resnet/estimator_benchmark.py [hunk 2]
```diff
@@ -0,0 +1,380 @@
...
+CIFAR10_RESNET56_MAX_TOP_1_ACCURACY = 0.938
+IMAGENET_DATA_DIR_NAME = 'imagenet'
+CIFAR_DATA_DIR_NAME = 'cifar-10-batches-bin'
+FLAGS = flags.FLAGS
+
+
+class EstimatorBenchmark(tf.test.Benchmark):
+  """Base benchmark class for estimator based benchmarks."""
+
+  local_flags = None
```
Comment (reedwm) [inline official/resnet/estimator_benchmark.py]: Can you give a comment explaining what this is?

File: official/resnet/estimator_benchmark.py [hunk 3]
```diff
@@ -0,0 +1,380 @@
...
+    FLAGS.dtype = 'fp32'
+    FLAGS.hooks = ['ExamplesPerSecondHook']
+    self._run_and_report_benchmark()
+
+  def benchmark_graph_fp16_8_gpu(self):
+    """Test FP16 8 GPUs graph mode."""
+    self._setup()
+    FLAGS.num_gpus = 8
+    FLAGS.data_dir = self.data_dir
+    FLAGS.batch_size = 128 * 8
```
Comment (reedwm) [inline official/resnet/estimator_benchmark.py]: Not 256 * 8?

**Comment by tfboyd** [comment]
@reedwm   OK I think I greatly improved the docstrings.

**Pull request merged**